test/014-request.test.js

```diff
@@ -77,4 +90,82 @@ describe('REQUEST TEST SUITE', () => {
           .to.equal('You are already a mentor');
       });
   });
+
+  describe('RESOLVE REQUEST', () => {
+    it('Should not allow resolve when request is not found',
+      async () => {
+        const response = await chai
+          .request(server)
+          .patch(`${baseUrl}/approve/${fakeId}`)
+          .set('Authorization', token);
+        expect(response.body.error)
+          .to.equal('Request not found');
+      });
+
+    it('Only an admin should be allowed to resolve request',
+      async () => {
+        const response = await chai
+          .request(server)
+          .patch(`${baseUrl}/approve/${id}`)
+          .set('Authorization', token);
+        expect(response.body.error)
+          .to.equal('Unauthorized');
+      });
+
+    it('if request is found Admin should be able to resolve request',
+      async () => {
+        const response = await chai
+          .request(server)
+          .patch(`${baseUrl}/approve/${id}`)
+          .set('Authorization', adminToken);
+        expect(response.status).to.equal(200);
+      });
+
+    it('should not resolve request if uuid is invalid',
+      async () => {
+        const response = await chai
+          .request(server)
+          .patch(`${baseUrl}/approve/${5}`)
```

Comment:
No need for a template string here if you are putting `5` for the id param.

Reply:
ok,.. noted